### PR TITLE
Remove projections from ionosphere and conductingsphere perturbed B computations

### PR DESCRIPTION
**Pull request by ykempf**
So this is the working state. I guess we can even ditch the whole projection functions now that no boundary is using that?

### CODE REVIEW

**Comment by ykempf** [comment]
If we agree on this, shall we ditch the projection function altogether? Also, shall we rename conductingsphere to... something else?

**Comment by ykempf** [comment]
That might break the testpackage diffing? Let's see.

File: sysboundary/copysphere.cpp [hunk 1]
```diff
@@ -40,65 +40,65 @@
...
-         Readparameters::add(pop + "_conductingsphere.VY0", "Bulk velocity of conductospheric distribution function in X direction (m/s)", 0.0);
-         Readparameters::add(pop + "_conductingsphere.VZ0", "Bulk velocity of conductospheric distribution function in X direction (m/s)", 0.0);
-         Readparameters::add(pop + "_conductingsphere.fluffiness", "Inertia of boundary smoothing when copying neighbour's moments and velocity distributions (0=completely constant boundaries, 1=neighbours are interpolated immediately).", 0);
+         Readparameters::add(pop + "_copysphere.rho", "Number density of the copysphere (m^-3)", 0.0);
+         Readparameters::add(pop + "_copysphere.T", "Temperature of the copysphere (K)", 0.0);
+         Readparameters::add(pop + "_copysphere.VX0", "Bulk velocity of conductospheric distribution function in X direction (m/s)", 0.0);
+         Readparameters::add(pop + "_copysphere.VY0", "Bulk velocity of conductospheric distribution function in X direction (m/s)", 0.0);
+         Readparameters::add(pop + "_copysphere.VZ0", "Bulk velocity of conductospheric distribution function in X direction (m/s)", 0.0);
```
Comment (ursg) [inline sysboundary/copysphere.cpp:71]: These are still saying "conductospheric"

**Comment by ykempf** [comment]
We're good with this one I suppose?

**Review by ursg — APPROVED** [review]
Yup

**Comment by markusbattarbee** [comment]
Diffs exist in conducting/copysphere simulations but are expected due to a change to the boundary condition handling. Can't find any references to conductingsphere via grepping the branch. I guess it's good indeed.

**Pull request merged**